### DISCUSSION TITLE
[Idea] Computer description

### DISCUSSION THREAD

**Discussion by buggzbeny** [glpi-project/glpi-agent#484] [Ideas]
### Your idea

Hello,
Fusion inventory agent used in OS.pm	:

my $description = getRegistryValue(path => 'HKEY_LOCAL_MACHINE/SYSTEM/CurrentControlSet/Services/lanmanserver/Parameters/srvcomment');

with that we can add into inventory description of computer.

This no longer available in GLPI agent. Can you add it ?

Or how can i do to use this information with glpi agent ?

thanks

**Comment by g-bougard** [comment]
Hi @buggzbeny 
as far I know, this is still supported agent-side:
https://github.com/glpi-project/glpi-agent/blob/ff26ea022796d0a618894789241b5557e6c28fbe/lib/GLPI/Agent/Task/Inventory/Win32/Hardware.pm#L57

Can you try to generate the hardware inventory with this command from an admin console and from the installation folder ?
```
glpi-inventory --partial hardware
```

If you have the expected description, the problem may be on server-side ? Which glpi version are you using ?

**Comment by buggzbeny** [comment]
Hello,

from cmd on the computer : 

"hardware": {
         "chassis_type": "Other",
         **"description": "salle-test",**
         "memory": 6143,
         
         
i have description i need, but i can t find this information on GLPI server. I'm on GLPI 10.0.6 with GLPI Agent 1.5

**Reply by g-bougard** [reply]
I think this is just not implemented in GLPI. You should open an issue on glpi project providing an example inventory, even partial (you can remove too sensible keys from the json, but test to import it manually from admin -> inventory to be sure it contains required datas).
